### DISCUSSION TITLE
How to setup Unit Testing Framework?

### DISCUSSION THREAD

**Discussion by whittenator** [AxisCommunications/acap-native-sdk-examples#157] [Q&A]
Is there any examples or advice someone can give on how to setup a Unit Testing framework like Google Test? Where you could potentially have the Unit Tests be ran on the Development Docker with the hosts architecture?

Thanks

**Comment by pataxis** [comment]
Hi @whittenator , from the ACAP SDK side we don't have any unit test examples today.
